src/site/filters/liquid.js

```diff
@@ -944,8 +944,9 @@ module.exports = function registerFilters() {
       formattedTitle = `${formattedTitle} | Veterans Affairs`;
     }
 
-    // Remove ' | | ' from the title.
+    // Remove ' | | ' and ' |  | ' from the title.
     formattedTitle = formattedTitle?.replace(' | | ', ' | ');
+    formattedTitle = formattedTitle?.replace(' |  | ', ' | ');
```

Comment:
```suggestion
    formattedTitle = formattedTitle?.replace(/\s*\|\s*\|/, ' | ');
```
<img width="631" alt="Screen Shot 2021-07-15 at 11 06 46 AM" src="https://user-images.githubusercontent.com/3144003/125811311-05c59208-7ad1-4166-a121-5d6ecfe7258a.png">